src/interface/task.interface.ts

```diff
@@ -0,0 +1,13 @@
+export enum taskStatus {
+  inProgress = 'IN PROGRESS',
```

Comment:
For constant values, it would be better to have it as `IN_PROGRESS='IN_PROGRESS'`